docs/application/native/guides/text-input/multi-assistant.md

```diff
@@ -0,0 +1,444 @@
+# Multi-assistant
+
+
+Multi-assistant allows the user to install and use multiple voice assistants on the same machine.
+
+The Multi-assistant framework manages voice assistants that are built with Multi-assistant client library and activates one among them when either the wake word is detected or the voice key is pressed.
+
+The main features of the Multi-assistant API include:
+
+- Retrieving information
+
+  You can [get various information](#info) from the Multi-assistant framework:
+
+  - Client state
+
+    The client state is changed by functions and applied as a precondition for each API.
+
+  - Service state
+
+    The service state is changed when the user starts or stops interacting with the installed voice assistants.
+
+  - Active state
+
+    Active state indicates whether the current client application is expected to serve as an active voice assistant for the user or not. An active voice assistant is responsible for responding to the user utterance and must provide appropriate feedback.
+
+  - Audio Data
+
+    When the assistant gets activated, it retrieves the user utterance audio data from the Multi-assistant framework in order to process user's voice command.
+
+- Updating result information
+
+  - ASR result
+
+    To provide visual feedback while processing the voice command, text representation that are converted from audio data needs to be updated as frequently as possible before the final result is retrieved.
+
+  - Result
+
+    Represents the final response to the user's query, including the summarized text information that will be displayed on the Multi-assistant panel if the device supports it.
+
+  - Recognition Result
+
+    Specifies whether the processing was successful or not.
+
+- Sending requests
+
+  - Adjust background volume
+
+When a voice assistant starts playing the voice feedback, it can request to adjust the background volume for making the voice feedback more audible.
```

Comment:
Please add four spaces to fix the alignment:

- Adjust background volume

    When a voice assistant starts playing the voice feedback, it can request to adjust the background volume for making the voice feedback more audible.

Reply:
Suggested changes uploaded : 2c1a24a1e02be6d3d3b9fd1b0a1fcfdae4c07534